share/spack/gitlab/cloud_pipelines/stacks/developer-tools-darwin/spack.yaml

```diff
@@ -0,0 +1,82 @@
+spack:
+  view: false
+  packages:
+    all:
+      require:
+      - target=aarch64
+  concretizer:
+    unify: true
```

Comment:
If you want things to rebuild when hashes change, I think you want `reuse: false` here too.

Reply:
Thanks for that, it was wrong on the linux side as well.

Reply:
I think `unify: true` was correct both here and in the linux side. That just means that the environment has a single configuration for each "runtime" package, and can be projected to a standard `/bin`, `/lib` etc. layout without conflicts.

Using `unify: false` we open the door for different specs to use slightly different library dependencies, since the optimization is done spec by spec, instead on the environment as a whole.

Reply:
Fwiw I posted a comment referring to `unify: false`, I saw later that we are discussing `reuse: false` - which is not in the diff. Deleted my previous comment as it was just noise.